From: Nathan Phillip Brink <binki@gentoo.org>
Date: 2011-07-08
Subject: Initialize gobject-types in dbus-plugin, fixes segfault when
	GTK+ is disabled (and thus g_type_init() isn't implicitly
	called by some gtk+ initialization function.)

--- a/src/common/dbus/dbus-plugin.c
+++ b/src/common/dbus/dbus-plugin.c
@@ -889,6 +889,8 @@
 	guint request_name_result;
 	GError *error = NULL;
 
+	g_type_init();
+
 	dbus_g_object_type_install_info (REMOTE_TYPE_OBJECT,
 					 &dbus_glib_remote_object_object_info);
 
